selftests/unit/test_utils_cpu.py

```diff
@@ -11,8 +12,12 @@
 
 
 def recent_mock():
-    major = int(mock.__version__.split('.')[0])
-    return major >= 2
+    if sys.version_info[0] < 3:
+        major = int(mock.__version__.split('.')[0])
+        return major >= 2
+    elif sys.version_info[0] == 3 and sys.version_info[1] < 6:
```

Comment:
So recent mock is ">2" on py2 or py3.6+ or python4+, right? Well it looks quite magic, a comment with the required features would help in the future, what do you think?

Reply:
@ldoktor actually, I think this may be simpler... and the PR itself needs more work.  You can leave this AS IS for now.  I'm closing it.